trino/client.py

```diff
@@ -218,6 +228,12 @@ def get_session_property_values(headers, header):
         for k, v in (kv.split("=", 1) for kv in kvs)
     ]
 
+def get_prepared_statement_values(headers, header):
```

Comment:
this is copy of the method above this with quoting method replaced. This will be unified as follow-up.

Note that there are quoting bugs pre-exisiting for session properties for example - I'll add tests and fix them in separate PR.

Reply:
In the follow-up we can add more meaningful var names. I understand key-values nomenclature but would prefer something explicit.

Reply:
Agreed. Plan is to have a helper module/module functions to assist with reading:

- value of a header by name
- values of a "collection" header
- key-value from a map-type "collection" header

Current we have one such function for each header (roles, prepared statements, session properties).